ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -184,8 +189,11 @@ def adaptive_temporary_interface(vm_set_name, interface_name, reserved_space=0):
 class HostInterfaces(object):
     """Data descriptor that supports multi-DUTs interface definition."""
 
-    def __get__(self, obj, objtype):
-        return obj._host_interfaces
+    def __set_name__(self, owner, name):
```

Comment:
`__set_name__` is introduced by Python 3.6, it doesn't work for Python2

Reply:
Nice catch! I just realized that I only tested on Ubuntu20 which uses python3.